### PR TITLE
OAL supports generating metrics from events

### LINKED ISSUE #7126: [DiSCUSSION] OAL supports generating metrics from events
It's kind of doubtful at the first thought of letting OAL support events, but I lately believe it's helpful for some events to be supported by OAL, especially in Kubernetes events, such that we can have metrics to answer the questions like

- How often pods cannot be scheduled? Are they related to capacity errors?
- What is the distribution of Pods, Deployments Created / Updated through out the day?
- Are there significant amount of "Error" events right now?

@wu-sheng what do you think of this

### PR DESCRIPTION
**Pull request by kezhenxu94**
- [x] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #<issue number>. Closes #7126 
- [x] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/CHANGES.md).

### CODE REVIEW

File: oap-server/server-bootstrap/src/main/resources/oal/event.oal [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+ *
+ */
+
+event_total = from(Event.*).count();
+
+event_error_count = from(Event.*).filter(type == "Error").count();
+event_normal_count = from(Event.*).filter(type == "Normal").count();
+
+event_start_count = from(Event.*).filter(name == "Start").count();
+event_shutdown_count = from(Event.*).filter(name == "Shutdown").count();
```
[the remark this answers is not in the document]
Comment (kezhenxu94) [reply oap-server/server-bootstrap/src/main/resources/oal/event.oal:25]: > Where are we going to use these metrics? Would there be a new dashboard? I can't see ui-templates change.

Yes, added a new dashboard

[the remark this answers is not in the document]
Comment (wu-sheng) [reply oap-server/server-bootstrap/src/main/resources/oal/event.oal:25]: Please provide a screenshot.

[the remark this answers is not in the document]
Comment (kezhenxu94) [reply oap-server/server-bootstrap/src/main/resources/oal/event.oal:25]: Screen Shot 2021-06-19 at 12 31 45

**Comment by wu-sheng** [comment]
I think `event.md` should be updated about 2 parts
1. We will visualize the event in the dashboard if entity matched. Maybe a little update about UI introduction? http://skywalking.apache.org/docs/main/latest/en/ui/readme/
2. We need to document what metrics we provide in the dashboard for the event.

**Comment by kezhenxu94** [comment]
> I think `event.md` should be updated about 2 parts
> 
> 1. We will visualize the event in the dashboard if entity matched. Maybe a little update about UI introduction? http://skywalking.apache.org/docs/main/latest/en/ui/readme/
> 2. We need to document what metrics we provide in the dashboard for the event.

Added a section of event metrics

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**